README.md

```diff
@@ -26,6 +26,39 @@ The following event types (incl. their subtypes) are supported and can be used t
 
 For further details and naming see the [vSphere Web Services API](https://code.vmware.com/apis/358/vsphere#/doc/vim.event.Event.html) documentation.
 
+## Other configuration
+
+### Credentials
+
+You can pass credentials via arguments (not recommended).
+
+```sh
+./vcenter-connector \
+  -vc-user="admin" \
+  -vc-pass="test1234" \
+```
+
+Or use a secret and pass the name:
+
+```sh
+./vcenter-connector \
+  -vc-user="" \
+  -vc-pass="" \
+  -vc-user-secret-name=vcenter-username \
+  -vc-password-secret-name=vcenter-password
+```
+
+Use `kubectl` to create the secrets you need ahead of time in the namespace where you deploy the connector.
+
+```sh
+kubectl create secret generic vcenter-secrets \
+  -n openfaas \
+  --from-literal vcenter-username=user \
+  --from-literal vcenter-password=pass
+```
+
+Now mount your secret at `/var/openfaas/secrets/` in your Kubernetes Deployment YAML file.
```

Comment:
This description is somewhat confusing for the reader since the `connector-dep.yaml` in this PR uses `/var/secrets` (overwritten with `secret_mount_path`). If a user forgets to set this env variable and uses the `mountPath` from the example, this will fail.

Perhaps we can add more context here:

Now mount your secret at `/var/openfaas/secrets/` in your Kubernetes Deployment YAML file. This location can be overwritten with the `secret_mount_path` environment variable in the Kubernetes deployment manifest. For an example, see `yaml/kubernetes/connector-dep.yml`.

Reply:
I agree, it should be done with helm and templating.

I wish I could throw money at this, but I don't have a customer at the moment.